ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,254 @@
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('ABC123');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('ABC123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await page.getByRole('button', { name: 'subscript' }).click();
+  });
+
+  test('Text tool -  Applying styles - Superscript', async ({ page }) => {
+    // Test case: EPMLSOPKET-2259
+    // Checking if possible to put Superscript style on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('ABC123');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'superscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('ABC123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'superscript' }).click();
+    await page.getByRole('button', { name: 'superscript' }).click();
+  });
+
+  test('Text tool - Applying styles - Combination of styles', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-2260
+    // Checking if possible to put different styles on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST123');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('25', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await page.getByRole('button', { name: 'italic' }).click();
+    await page.getByRole('button', { name: 'superscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await pressButton(page, 'Apply');
+  });
+
+  test('Text tool - Save as .ket file', async ({ page }) => {
+    // Test case: EPMLSOPKET-2235
+    // Checking if possible to put different styles on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST321');
+    await pressButton(page, 'Apply');
+    await page.getByText('TEST321').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('20', { exact: true }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await saveToFile('ketfile01.ket', 'ketFile');
```

Comment:
If we want to check the saving of a file, it is better to do this as a separate test.

Reply:
As I wrote earlier, it is better to divide the test into two. We won’t check whether the file is saved this way.